### PR TITLE
Mis 7

### PR DESCRIPTION
**Pull request by Leo24**
Added fixDate check for dates collected from forms

### CODE REVIEW

**Comment by tmccormi** [comment]
Is there a reason you are closing the pull requests before we review them?

**Comment by Leo24** [comment]
Yes, I have issues in squashing commits with rebase.
I did not do it in right way, so in branches left several commits instead one.
Sorry for disturbing you with emails, I'll do squashing commits in the nearest future, so you can check it.

**Comment by tmccormi** [comment]
OK.   We can help you with that if necessary, it can be tricky

Tony McCormick, CTO
www.mi-squared.com
Support: 866-735-0897, Direct: 713-574-6709
My Calendar:  http://bit.ly/XznvDo
"Genuinely evolved interfaces are transparent, so transparent as to be
invisible" - William Gibson



On Fri, May 20, 2016 at 12:01 PM, Leonid <notifications@github.com> wrote:

> Yes, I have issues in squashing commits with rebase.
> I did not do it in right way, so in branches left several commits instead
> one.
> Sorry for disturbing you with emails, I'll do squashing commits in the
> nearest future, so you can check it.
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/mi-squared/openemr/pull/10#issuecomment-220691520>
>

-- 
Please be aware that e-mail communication can be intercepted in 
transmission or misdirected. Please consider communicating any sensitive 
information by telephone.  The information contained in this message may be 
privileged and confidential. If you are NOT the intended recipient, please 
notify the sender immediately with a copy to hipaa-security@mrsb-ltd.com and 
destroy this message.

**Comment by Leo24** [comment]
I have figured out how to squash commits, thanks for understanding.
Could you, please, provide some clarifications about git flow.
I cloned repository, than changed sql configs for local development and pass them in "Innitial commit", 
into master branch. Then I created new branch "MIS-6" for task did work and commited changes.
But "Innitial commit" came into PR of branch "MIS-6".
Tryed to cherry-pick "Innitial commit", to another branch and then push data to remote, but unsecessfully.
How I must I, in future, start work on new projects to prevent getting local changes to remote?

File: sites/default/sqlconf.php [hunk 1]
```diff
@@ -1,30 +1,30 @@
...
 $sqlconf["login"] = $login;
 $sqlconf["pass"] = $pass;
 $sqlconf["dbase"] = $dbase;
-
 //////////////////////////
 //////////////////////////
 //////////////////////////
 //////DO NOT TOUCH THIS///
-$config = 0; /////////////
+$config = 1; /////////////
```
Comment (rachoac) [inline sites/default/sqlconf.php]: Probably should reverse this -- it's says do not touch

File: sites/default/sqlconf.php [hunk 2]
```diff
@@ -1,30 +1,30 @@
...
-//  Referenced from /library/sqlconf.php.
-
-global $disable_utf8_flag;
-$disable_utf8_flag = false;
 
 $host	= 'localhost';
 $port	= '3306';
 $login	= 'openemr';
-$pass	= 'openemr';
+$pass	= 'root';
```
Comment (rachoac) [inline sites/default/sqlconf.php]: Any reason we are changing default password

Comment (tmccormi) [inline sites/default/sqlconf.php]: This needs to be reverted as well.

**Pull request closed without merging**